### DISCUSSION TITLE
Change Target Folder for Different Profiles?

### DISCUSSION THREAD

**Discussion by purpleemoji** [UltimaHoarder/UltimaScraper#1344] [General]
Sorry if this has been asked before.

I just so happen to use three different OF accounts (don't ask ;) ) and was wondering if I can modify the config.json file to select a target directory  based on which profile is being scraped. Basically I would like to have it be conditional folder selection like .sites.[username] based on which username is selected  after the "Choose Profile" prompt.

I'm not a coder so I'm currently just editing my config file using a search and replace depending on which profile I happen to be scraping. 

As always, a million thanks to digitalcriminal for this awesome tool!!

**Comment by UltimaHoarder** [comment]
I've added {profile_username} to the path format options, so download the latest commit and thanks :)

**Comment by purpleemoji** [comment]
Awesome. Thanks for implementing this.
